.pipelines/unit-tests-template.yml

```diff
@@ -12,5 +12,7 @@ jobs:
         displayName: Build
       - script: make unit-test
         displayName: Run unit tests
+      - script: make lint
```

Comment:
Could you increase the timeout from 10 min to maybe 20 minutes? Looks like it timed out after 10 minutes.

Reply:
Sure, just wanted to point out that the 10m cap applies to the job and not the step. The current run failed after about 6min.